packages/liveblocks-react/src/room.tsx

```diff
@@ -108,11 +103,18 @@ import type {
   UseStorageStatusOptions,
   UseThreadsOptions,
 } from "./types";
+import type { UmbrellaStore, UmbrellaStoreState } from "./umbrella-store";
+import {
+  addReaction,
+  deleteComment,
+  removeReaction,
+  upsertComment,
+} from "./umbrella-store";
 import { useScrollToCommentOnLoadEffect } from "./use-scroll-to-comment-on-load-effect";
 
 const SMOOTH_DELAY = 1000;
 
-const noop = () => { };
+const noop = () => {};
```

Comment:
@jrowny I noticed this as well on your PR, but our linters somehow keep fighting over these changes. I think this only happens if we have different versions of Prettier (or ESLint maybe) installed. Can you make sure your VSCode is configured to use the ESLint and Prettier versions from this repo (the version in `node_modules`), and not maybe some globally installed different version of ESLint or Prettier?

![](https://github.com/user-attachments/assets/be674065-d4b3-467d-a03c-1fd0155615fd)

The way to verify is to run `npm ci` in the root, and then run `turbo run format` on this package and there should not be any changes.